localnet/kubernetes/values-relayminer.yaml

```diff
@@ -1,4 +1,3 @@
 config:
   query_node_url: tcp://sequencer-poktroll-sequencer:36657
   network_node_url: tcp://sequencer-poktroll-sequencer:36657
-  
```

Comment:
Is this change intentional or automatic (IDE)? I think we should prefer trailing new-lines in our files.

Reply:
Should be back.

Reply:
Doesn't look like it. Did you `git checkout main -- localnet/kubernetes/values-relayminer.yaml`?

Reply:
Doesn't look like it. Did you `git checkout main -- localnet/kubernetes/values-relayminer.yaml`?